src/dbsync/schema.rs

```diff
@@ -1,17 +1,24 @@
 pub mod sql_types {
-    #[derive(diesel::sql_types::SqlType)]
+    use diesel::{
+        sql_types::SqlType,
+        query_builder::QueryId
+    };
+
+    use diesel_derive_enum::DbEnum;
+
+    #[derive(SqlType, QueryId)]
     #[diesel(postgres_type(name = "Syncstatetype"))]
     pub struct Syncstatetype;
 
-    #[derive(diesel::sql_types::SqlType)]
+    #[derive(SqlType, QueryId)]
     #[diesel(postgres_type(name = "Scriptpurposetype"))]
     pub struct Scriptpurposetype;
 
-    #[derive(diesel::sql_types::SqlType)]
-    #[diesel(postgres_type(name = "Rewardtype"))]
-    pub struct Rewardtype;
+    #[derive(SqlType, QueryId)]
+    #[diesel(postgres_type(name = "rewardtype"))]
+    pub struct RewardTypeEnum;
```

Comment:
The `Enum` suffix is redundant and misleading for a type that is defines as `struct`. Please remove it.

Reply:
it represents enum in the database so I don't think it's misleading. Also helps distinguish that item from the other item called RewardType in `models.rs`

Reply:
Putting `Enum`,  `Struct` etc in types name is not a best practice. It's redundant, and does not improve anything, the  define a type as `enum` is self explanatory. Please remove it.

Reply:
I think that rule is just a rule of thumb. in our case we have a good reason for putting the word "Enum" in there because it's not obvious from the syntax that it's an enum. The syntax says the word "struct" but it's actually not a struct, but rather an SQL enum. It's not a Rust enum, but rather SQL enum. Struct is used as a base on which to place attributes that convert it into SQL enum